pkg/server/policy/policy.go

```diff
@@ -0,0 +1,170 @@
+package policy
```

Comment:
I'm wondering if the directory name `policy` is specific enough, thinking that we may end up having other kind of _policies_ in the future that may have a collision with this. Maybe `authpolicy` sounds better to me. @azdagron Thoughts?

Reply:
authpolicy seems nice :)